### PR TITLE
collect instantly using prime coupon

### PR DESCRIPTION
**Pull request by anjeshshrestha**
need to test

### CODE REVIEW

File: src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java [hunk 1]
```diff
@@ -45,6 +45,9 @@ public boolean update(int expiryTime) {
     public boolean hireRetriever(Retriever retriever) {
         if (data.getAvailableSlots() <= 0) return false;
         //retriever.getCost(); // TODO: check cost
+        if(retriever.getCreditCost() > main.statsManager.credits || retriever.getUridiumCost() > main.statsManager.uridium || retriever.getPermitCost() > data.getPermit()){
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java:48]: split line, it's too long
```suggestion
        if (retriever.getCreditCost() > main.statsManager.credits || 
                retriever.getUridiumCost() > main.statsManager.uridium || 
                retriever.getPermitCost() > data.getPermit()) {
```

File: src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java [hunk 2]
```diff
@@ -58,6 +61,26 @@ public boolean hireRetriever(Retriever retriever) {
         return false;
     }
 
+    public boolean collectInstant(InProgress progress) {
+        try {
+            System.out.println("Collecting Instant: Slot " + progress.getSlotId());
+            if(data.getPrimeCoupons() == 0) return handleResponse("Instant Collect Failed", progress.getId(), "No Prime Coupon Available For Instant Collection");
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java:67]:
```suggestion
            if (data.getPrimeCoupons() == 0) 
                return handleResponse("Instant Collect Failed", progress.getId(), 
                        "No Prime Coupon Available For Instant Collection");
```

**Review by Pablete1234 — CHANGES_REQUESTED** [review]
Just a few format changes

Have you tested this, does it work as expected?

**Comment by anjeshshrestha** [comment]
did test today, seems to be working good

**Pull request merged**